app/uploaders/achievement_badge_uploader.rb

```diff
@@ -0,0 +1,13 @@
+# Image uploader for Achievement Badge
+# This class contains achievement badge specific settings for the image uploader.
+class AchievementBadgeUploader < ImageUploader
```

Comment:
Actually I would expect people to use ImageUploader directly, but I think what @weiqingtoh did here is mainly for supporting a `default_url`, it's still OK.

They other way was handle the default url in helpers as what `User#icon` does, or just upload a default icon when achievement is created, so that we don't need a separate achievement uploader.